packages/templates/src/raw/12-hours-in-barcelona/template.json

```diff
@@ -373,39 +289,21 @@
           "groupId": "68836b5c-a442-4407-a938-a790f564d9de"
         }
       ],
-      "backgroundOverlay": "none",
```

Comment:
Here's an example of the lingering `backgroundOverlay` property.
`v0023_convertOverlayPattern.js` should have been taking care of this.